package-lock.json

```diff
@@ -3159,6 +3159,11 @@
         }
       }
     },
+    "moment": {
```

Comment:
Please remove this package also.

Reply:
Done in 7e2091b9ee648b876c20c7d49035ba51e455e7f5